exchanges/order/order_types.go

```diff
@@ -370,6 +379,7 @@ const (
 	Chase           // chase limit order
 	OptimalLimit
 	MarketMakerProtection
+	Bracket // Sets both a profit target and a stop loss simultaneously.
```

Comment:
🚧 We already have `StopLimit` and `StopMarket` types, which is a synonym for Bracket order.
So I think that this is about treating the text representation of `StopLimit` as `BRACKET` for Coinbase, maybe?
```suggestion
	Bracket // Sets both a profit target and a stop loss simultaneously.
```